Client/core/CSettings.cpp

```diff
@@ -3609,12 +3631,16 @@ void CSettings::SaveData()
         CVARS_SET("update_auto_install", iSelected);
     }
 
-    // Map alpha
+    // Radar map alpha
```

Comment:
`Radar map` can be misinterpreted with SA radar

Reply:
Radar map is the f11 map. It's what it's called everywhere in the code unfortunately, it's not the minimal :/ I could refactor the naming everywhere for something like FullMap or BigMap